pkg/integration/components/assertion_helper.go

```diff
@@ -11,9 +11,9 @@ type assertionHelper struct {
 }
 
 // milliseconds we'll wait when an assertion fails.
-var retryWaitTimes = []int{0, 1, 1, 1, 1, 1, 5, 10, 20, 40, 100, 200, 500, 1000, 2000, 4000}
+var retryWaitTimes = []int{0, 1, 1, 1, 1, 1, 5, 10, 20, 40, 100}
```

Comment:
Thanks for this! 😄  (I was always wondering why we have to wait so long.)

Reply:
Ah actually that was accidentally committed: I was reducing the timeout for my own local usage. But if CI can handle it then I'm fine to keep it that way across the board

Reply:
Ah. 😄 

Maybe we could have an environment variable that is only set on CI and that makes it wait longer.